website/utils/url_utils.ts

```diff
@@ -303,8 +315,8 @@ const tryDecodeURLList = (url: string, delim = ","): string[] | undefined => {
  * a key-value object.
  *
  * @param data The string to parse. Expected to be in the format
- * "key1:value1,key2:value2,...". Colons and commas in keys or values
- * should be encoded using `encodeURIComponent`.
+ * "key1:value1,key2:value2,...". Commas in keys or values
+ * must be encoded using `encodeURIComponent`.
```

Comment:
Possible bug here where commas can be decoded by ShareModal after being encoded in values. Maybe just explicitly forbid commas

Reply:
This will be fine actually, since URLParams does one level of encoding